test/stress/testdata/eventbus/jetstream.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventBus
+metadata:
+  name: default
+spec:
+  jetstream:
+    version: 2.7.4
```

Comment:
I suppose this version number is very specific (is that an issue?)

Reply:
I think we can have a `latest` tag.